requirements/cext_noarch.txt

```diff
@@ -0,0 +1,4 @@
+# These are unique requirements for C extensions, but the
+# ones that don't need to be shipped in one version per
+# arch/OS/pyVesion
+pyOpenSSL==17.5.0
```

Comment:
I think 17.5.0 will not work for us currently, since it requires `cryptography` to be >= 2.1.4. But we are using 2.0.3 right now. I tried yesterday to upgrade `pyopenssl` to 17.5.0, and there were some errors saying that "attribute xxx not found". So I think for now, it will be better to use `pyopenssl` 17.4.0 to be compatible with the current version of `cryptography` we are using?

Reply:
Thanks! Will do that right meow!

Yes, the upgrade path for `cryptography` starts by adding the 2.1.4 version to Morango's test matrix, and then we can upgrade it in Kolibri... so this has to wait.